application/fin/netsuite_test.go

```diff
@@ -100,16 +96,12 @@ func TestNetSuite_Export(t *testing.T) {
 		body, err := io.ReadAll(contents)
 		require.NoError(t, err)
 
-		want := transaction1Row
+		// don't try to compare the row number since we can't guarantee the transaction batch ordering
+		want := transaction1Row[12:]
 		if name == "bar" {
-			want = transaction2Row
+			want = transaction2Row[12:]
 		}
 
-		// Replace the first instance of "<number>," with "\d,"
-		// So that map order will not affect test
-		// For example, 2023080000001 becomes 202308000000\d
-		want = re.ReplaceAllString(want, `$1\d,$2`)
-
-		require.Regexp(t, netSuiteHeader+want, string(body))
+		require.Contains(t, string(body), want)
```

Comment:
This seems a bit cleaner, and shouldn't break if we decide to change csv contents later (assuming we keep id as the first field).

```suggestion
		// don't try to compare the row number since we can't guarantee the transaction batch ordering
		row := transaction1Row
		if name == "bar" {
			row = transaction2Row
		}
		_, want, _ := strings.Cut(row, ",")

		require.Contains(t, string(body), want)
```